mlir-sycl/lib/Dialect/IR/SYCLOps.cpp

```diff
@@ -64,11 +64,8 @@ bool SYCLCastOp::areCastCompatible(TypeRange Inputs, TypeRange Outputs) {
 }
 
 LogicalResult SYCLConstructorOp::verify() {
-  if (getNumOperands() == 0)
-    return emitOpError("A sycl::constructor must be passed the object to build "
-                       "as the first argument");
```

Comment:
lit test `invalid.mlir` fails because this is removed.

Reply:
Yep, we should remove that test (not applicable anymore)

Reply:
removed